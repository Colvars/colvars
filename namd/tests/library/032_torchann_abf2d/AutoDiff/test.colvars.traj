# step         nn_phi                r_nn_phi               nn_psi                r_nn_psi              
           0   -2.66022205352783e+00 -2.66022205352783e+00   2.64080524444580e+00  2.64080524444580e+00 
           1   -2.65702581405640e+00 -2.66022317794724e+00   2.64224958419800e+00  2.64079391885964e+00 
           2   -2.65432882308960e+00 -2.66023364602892e+00   2.64337253570557e+00  2.64080198925227e+00 
           3   -2.65178489685059e+00 -2.66025795250247e+00   2.64434242248535e+00  2.64083652559973e+00 
           4   -2.64888143539429e+00 -2.66026345720680e+00   2.64515995979309e+00  2.64083374407879e+00 
           5   -2.64632439613342e+00 -2.66029609828320e+00   2.64512729644775e+00  2.64083937477918e+00 
           6   -2.64347410202026e+00 -2.66033537177595e+00   2.64515733718872e+00  2.64085131015421e+00 
           7   -2.64110517501831e+00 -2.66039894576593e+00   2.64455914497375e+00  2.64087395824469e+00 
           8   -2.63912129402161e+00 -2.66046144466070e+00   2.64341378211975e+00  2.64089484340015e+00 
           9   -2.63696074485779e+00 -2.66052698096927e+00   2.64169836044312e+00  2.64092641490024e+00 
          10   -2.63519239425659e+00 -2.66058722692556e+00   2.63923978805542e+00  2.64097602962568e+00 
          11   -2.63270568847656e+00 -2.66062970571351e+00   2.63662528991699e+00  2.64102106254642e+00 
          12   -2.63013505935669e+00 -2.66069154614378e+00   2.63314294815063e+00  2.64108061756652e+00 
          13   -2.62688398361206e+00 -2.66074570845387e+00   2.62913584709167e+00  2.64112999164946e+00 
          14   -2.62401294708252e+00 -2.66077916221273e+00   2.62441921234131e+00  2.64115208823163e+00 
          15   -2.62137413024902e+00 -2.66081044276743e+00   2.61924839019775e+00  2.64111847876899e+00 
          16   -2.61909818649292e+00 -2.66085256377206e+00   2.61365938186646e+00  2.64104761463810e+00 
          17   -2.61732101440430e+00 -2.66086766364394e+00   2.60709619522095e+00  2.64099573618114e+00 
          18   -2.61557674407959e+00 -2.66086828625528e+00   2.59994554519653e+00  2.64095373134199e+00 
          19   -2.61349749565125e+00 -2.66088450169815e+00   2.59253978729248e+00  2.64091907430937e+00 
          20   -2.61156010627747e+00 -2.66089137898683e+00   2.58498716354370e+00  2.64089678153854e+00 
