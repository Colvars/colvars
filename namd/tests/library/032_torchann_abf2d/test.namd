# alanine dipeptide
seed 1

# Variables
set outName         test

structure           ./ad-charmm36-vacuum.psf
coordinates         ./ad-charmm36-vacuum_eq.coor
parameters          ./par_all36m_prot.prm
paraTypeCharmm      on
temperature         300.0

## SIMULATION PARAMETERS                                  
hgroupcutoff        2.8
switching           on
switchdist          10.0
cutoff              12.0
pairlistdist        14.0
rigidbonds          none
exclude             scaled1-4
1-4scaling          1.0

# Integrator Parameters 
timestep            0.5 
nonbondedFreq       1 
fullElectFrequency  1 
stepspercycle       10
splitpatch          hydrogen
margin              2.0

# Output
outputenergies          1000
outputtiming            1000
restartfreq             100000
binaryoutput            no
binaryrestart           yes
outputname              $outName
restartname             $outName

# CONSTANT-T

langevin                on
langevinTemp            300.0
langevinDamping         10.0

# COM

commotion               no

colvars                 on
colvarsConfig           test.in

numSteps                20
