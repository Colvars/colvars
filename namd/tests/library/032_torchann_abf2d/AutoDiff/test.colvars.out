colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:   https://doi.org/10.1080/00268976.2013.813594
colvars: as well as all other papers listed below for individual features used.
colvars: SMP parallelism is enabled; if needed, use "smp off" to override this.
colvars: This version was built with the C++11 standard or higher.
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # sourceTclFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "nn_phi"
colvars:   Initializing a new "torchANN" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 6.2832
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 10 atoms requested: total mass = 132.079, total charge = -1.52.
colvars:     # model_file = "dihedrals_cpu.pt"
colvars:     # m_output_index = 0
colvars:     model loaded.
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.1
colvars:   # lowerBoundary = -3.3
colvars:   # upperBoundary = 3.3
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = on
colvars:   Enabling the extended Lagrangian term for colvar "nn_phi".
colvars:   # extendedTemp = 300 [default]
colvars:   # extendedFluctuation = 0.1
colvars:   Computed extended system force constant: 59.6157 [E]/U^2
colvars:   # extendedTimeConstant = 500
colvars:   Computed fictitious mass: 377521 [E]/(U/fs)^2   (U: colvar unit)
colvars:   # outputEnergy = off [default]
colvars:   # extendedLangevinDamping = 1 [default]
colvars:   # reflectingLowerBoundary = off [default]
colvars:   # reflectingUpperBoundary = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "nn_psi"
colvars:   Initializing a new "torchANN" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 6.2832
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 10 atoms requested: total mass = 132.079, total charge = -1.52.
colvars:     # model_file = "dihedrals_cpu.pt"
colvars:     # m_output_index = 1
colvars:     model loaded.
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.1
colvars:   # lowerBoundary = -3.3
colvars:   # upperBoundary = 3.3
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = on
colvars:   Enabling the extended Lagrangian term for colvar "nn_psi".
colvars:   # extendedTemp = 300 [default]
colvars:   # extendedFluctuation = 0.1
colvars:   Computed extended system force constant: 59.6157 [E]/U^2
colvars:   # extendedTimeConstant = 500
colvars:   Computed fictitious mass: 377521 [E]/(U/fs)^2   (U: colvar unit)
colvars:   # outputEnergy = off [default]
colvars:   # extendedLangevinDamping = 1 [default]
colvars:   # reflectingLowerBoundary = off [default]
colvars:   # reflectingUpperBoundary = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 2 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "abf" instance.
colvars:   # name = "abf1" [default]
colvars:   # colvars = { nn_phi, nn_psi }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # applyBias = on [default]
colvars:   # updateBias = on [default]
colvars:   # hideJacobian = off [default]
colvars:   Jacobian (geometric) forces will be included in reported free energy gradients.
colvars:   # fullSamples = 2000
colvars:   # inputPrefix =  [default]
colvars:   # historyFreq = 0 [default]
colvars:   # shared = off [default]
colvars:   # maxForce =  [default]
colvars:   # CZARestimator = on [default]
colvars:   # integrate = on [default]
colvars:   # UIestimator = off [default]
colvars:   Finished ABF setup.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (10 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: 
colvars: SUMMARY OF COLVARS FEATURES USED SO FAR AND THEIR CITATIONS:
colvars: 
colvars: - torchANN colvar component:
colvars:   Chen2022 https://doi.org/10.1021/acs.jcim.1c01010
colvars: 
colvars: - Colvars module:
colvars: - Colvars-NAMD interface:
colvars: - Optimal rotation via flexible fitting:
colvars:   Fiorin2013 https://doi.org/10.1080/00268976.2013.813594
colvars: 
colvars: - ABF colvar bias implementation:
colvars:   Henin2010 https://doi.org/10.1021/ct9004432
colvars: 
colvars: - Poisson integration of 2D/3D free energy surfaces:
colvars:   Henin2021 https://doi.org/10.1021/acs.jctc.1c00593
colvars: 
colvars: - CZAR eABF estimator:
colvars: - eABF implementation:
colvars:   Lesage2017 https://doi.org/10.1021/acs.jpcb.6b10055
colvars: 
colvars: - NAMD engine:
colvars:   Phillips2020 https://doi.org/10.1063/5.0014475
colvars: 
colvars: updating target temperature (T = 300 K).
colvars: Re-initialized atom group for variable "nn_phi":0/0. 10 atoms: total mass = 132.079, total charge = -1.52.
colvars: Re-initialized atom group for variable "nn_psi":0/0. 10 atoms: total mass = 132.079, total charge = -1.52.
colvars: The restart output state file will be "test.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars: Updating NAMD interface:
colvars: updating atomic data (10 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: updating target temperature (T = 300 K).
colvars: Re-initialized atom group for variable "nn_phi":0/0. 10 atoms: total mass = 132.079, total charge = -1.52.
colvars: Re-initialized atom group for variable "nn_psi":0/0. 10 atoms: total mass = 132.079, total charge = -1.52.
colvars: The restart output state file will be "test.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars:   Prepared sample and gradient buffers at step 0.
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.colvars.state".
colvars:   Integrated in 221 steps, error: 9.5063e-07
colvars:   Integrated in 221 steps, error: 9.5063e-07
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.colvars.state".
colvars:   Integrated in 217 steps, error: 9.22707e-07
colvars:   Integrated in 221 steps, error: 9.58126e-07
colvars: Saving collective variables state to "test.colvars.state".
